package.json

```diff
@@ -22,6 +22,8 @@
     "test-chrome": "sh scripts/testcafe-chrome.sh",
     "test-ie": "sh scripts/testcafe-ie.sh",
     "test-jest": "node scripts/test.unit.js --env=jsdom",
+    "test-ui": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe chrome ./**/ui.test.js",
+    "test-ui-hl": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe 'chrome:headless' ./**/ui.test.js",
```

Comment:
If you refactor it, please change also the testscript `test calls test-chrome, instead should call test-ui-hl imho` to use `test-ui-hl` instead of calling directly then `sh` script

Reply:
This is just for local testing and not meant to overrule the other scripts. But usually, when you develop, you have an instance of storybook running and then it's convenient when you run the tests on top without needing to fire up storybook again although it is already running.